.drone.yml

```diff
@@ -33,3 +33,6 @@ matrix:
   IMAGE_PHP:
     - fpfis/httpd-php-dev:5.6
     - fpfis/httpd-php-dev:7.1
+  COMPOSER_BOUNDARY:
```

Comment:
The matrix is extended with choices for the composer boundary, but this `${COMPOSER_BOUNDARY}` variable is not used in the pipeline, so it has no effect.

For an example on how to use this, see the pipeline of the Code Review component: https://github.com/openeuropa/code-review/blob/master/.drone.yml#L12

Reply:
👍

Reply:
I am not confortable with the syntax proposed in code-review
```
composer-install-lowest:
    group: prepare
    image: ${IMAGE_PHP=fpfis/httpd-php-dev:5.6}
    volumes:
      - /cache:/cache
    commands:
      - composer update --prefer-lowest
    when:
      matrix:
        COMPOSER_BOUNDARY: lowest
   composer-install-highest:
    group: prepare
    image: ${IMAGE_PHP=fpfis/httpd-php-dev:5.6}
    volumes:
      - /cache:/cache
    commands:
      - composer update
    when:
      matrix:
        COMPOSER_BOUNDARY: highest
[...]
  COMPOSER_BOUNDARY:
    - lowest
    - highest
```

I prefer the following syntax:
```
composer-install:
    group: prepare
    image: ${IMAGE_PHP=fpfis/httpd-php-dev:5.6}
    volumes:
      - /cache:/cache
    commands:
      - composer update ${COMPOSER_BOUNDARY}
[...]
  COMPOSER_BOUNDARY:
    - '--prefer-lowest'
    - ''
```

Reply:
It's better like this, I approve @voidtek .